doc/examples/debug/lib/main.dart

```diff
@@ -54,16 +57,16 @@ class MyGame extends BaseGame {
   bool debugMode() => true;
 
   void _start() {
-    AndroidComponent android = AndroidComponent();
+    final AndroidComponent android = AndroidComponent();
     android.x = 100;
     android.y = 400;
 
-    AndroidComponent android2 = AndroidComponent();
+    final AndroidComponent android2 = AndroidComponent();
     android2.x = 100;
     android2.y = 400;
     android2.yDirection = -1;
 
-    AndroidComponent android3 = AndroidComponent();
```

Comment:
The type here is not necessary (I know, it was me who originally wrote this)

Reply:
I don't mind the type, it makes it more well defined. It's the missing final that the linter complains ;)

Reply:
I realize that, but the type here is totally unnecessary, I am all for setting the type on method signatures, but on local attributions, where it is pretty clear what the type is, I say that we let the type inference work and we have less code written. But this is up to you.